src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -312,22 +345,35 @@ describe( 'ContentTabAssistant', () => {
 		);
 
 		render( <ContentTabAssistant selectedSite={ runningSite } /> );
-		expect( screen.getByText( 'Welcome to our service!' ) ).toBeVisible();
-		expect( screen.getByText( 'Initial message 1' ) ).toBeVisible();
-		expect( screen.getByText( 'Initial message 2' ) ).toBeVisible();
-		expect(
-			screen.getByText( 'This conversation is over two hours old.', { exact: false } )
-		).toBeVisible();
+
+		jest.advanceTimersByTime( MIMIC_CONVERSATION_DELAY + 1000 );
+		await waitFor(
+			() => {
+				expect( screen.getByText( 'Welcome to our service!' ) ).toBeVisible();
+				expect( screen.getByText( 'Initial message 1' ) ).toBeVisible();
+				expect( screen.getByText( 'Initial message 2' ) ).toBeVisible();
+				expect(
+					screen.getByText( 'This conversation is over two hours old.', { exact: false } )
+				).toBeVisible();
+			},
+			{ timeout: MIMIC_CONVERSATION_DELAY + 1000 }
+		);
 
 		( getIpcApi as jest.Mock ).mockReturnValue( {
 			showMessageBox: jest.fn().mockResolvedValue( { response: 0, checkboxChecked: false } ),
 		} );
-		fireEvent.click( screen.getByRole( 'button', { name: 'Clear the history' } ) );
-		await waitFor( () => {
-			expect( getIpcApi().showMessageBox ).toHaveBeenCalledTimes( 1 );
-			expect( screen.queryByText( 'Initial message 1' ) ).not.toBeInTheDocument();
-			expect( screen.queryByText( 'Initial message 2' ) ).not.toBeInTheDocument();
+		act( () => {
+			fireEvent.click( screen.getByRole( 'button', { name: 'Clear the history' } ) );
 		} );
+		jest.advanceTimersByTime( MIMIC_CONVERSATION_DELAY + 1000 );
```

Comment:
I think we don't need to advance timers as we haven't submitted a prompt but cleared the history.

Reply:
fixed